2021/scipy/README.md

```diff
@@ -0,0 +1,44 @@
+# SciPy 2021 Tutorial - _Fairness in AI Systems: From Social Context to Practice using Fairlearn_
```

Comment:
Can we have a more shallow path? How about: `2021-scipy-tutorial/`

@fairlearn/fairlearn-maintainers : opinions on this?
I don't expect we will have more than a few talks per year that we want to put in the repo?